### PR TITLE
Fix up docs link

### PR DESCRIPTION
**Pull request by alexlarsson**

### CODE REVIEW

**Comment by alexlarsson** [comment]
@rh-atomic-bot delegate=cgwalters

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ c440b3e

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ c440b3e

**Comment by cgwalters** [comment]
@alexlarsson Try doing the delegate again?

**Comment by alexlarsson** [comment]
@rh-atomic-bot delegate=cgwalters

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ c440b3e

File: ci/build.sh [hunk 1]
```diff
@@ -24,5 +24,13 @@ pkg_install_if_os fedora gjs parallel clang
  ostree --version
 )
 
+(git clone --depth=1 https://github.com/flatpak/flatpak/
+ cd flatpak
+ unset CFLAGS # the sanitizers require calling apps be linked too
+ build
+ make
```
Comment (cgwalters) [inline ci/build.sh]: `build` already invokes `make`.

File: ci/build.sh [hunk 2]
```diff
@@ -9,12 +9,12 @@ dn=$(dirname $0)
...
 pkg_install sudo which attr fuse \
     libubsan libasan libtsan \
-    elfutils ostree git \
+    elfutils ostree git gettext-devel \
     /usr/bin/{update-mime-database,update-desktop-database,gtk-update-icon-cache}
 pkg_install_if_os fedora gjs parallel clang
 
-# Temporarily build ostree from git master for https://github.com/flatpak/flatpak/pull/848
-(git clone --depth=1 https://github.com/ostreedev/ostree/
+# Use ostree minimal dependency
```
Comment (cgwalters) [inline ci/build.sh]: BTW see also https://github.com/projectatomic/papr/issues/10

**Comment by alexlarsson** [comment]
bot, retest

**Comment by alexlarsson** [comment]
@cgwalters do you have any idea why this installs ostree 2017.10 when the same in e.g. https://github.com/flatpak/flatpak/pull/1047 installes ostree 2017.11. Both enable updates-testing which should have the new ostree...

**Comment by alexlarsson** [comment]
@rh-atomic-bot r+ ee1f6b7

**Pull request closed without merging**